cmd-live.c

```diff
@@ -76,32 +76,7 @@ static bool can_skip_replay(struct opts *opts, int record_result)
 
 static void setup_child_environ(struct opts *opts)
 {
-	char buf[4096];
-	char *old_preload, *old_libpath, *libpath;
-
-	if (opts->lib_path) {
-		strcpy(buf, opts->lib_path);
-		strcat(buf, "/libmcount:");
-	} else {
-		/* to make strcat() work */
-		buf[0] = '\0';
-	}
-
-#ifdef INSTALL_LIB_PATH
-	strcat(buf, INSTALL_LIB_PATH);
-#endif
```

Comment:
Well, I think you don't use a full path unless `-L` option is given, right?  Then `LD_LIBRARY_PATH` is necessary for `INSTALL_LIB_PATH` IMO.

Reply:
You're right ! I'll change it